script/update-content-versions-and-check-for-archived-slugs.ts

```diff
@@ -0,0 +1,110 @@
+import fs from 'fs';
+import { isEqual } from 'lodash';
+import path from 'path';
+import { argv } from 'yargs';
+import { content } from '../src/app/content/routes';
+import { LinkedArchiveTreeNode } from '../src/app/content/types';
+import { flattenArchiveTree } from '../src/app/content/utils';
+import { makeUnifiedBookLoader } from '../src/app/content/utils';
+import { findArchiveTreeNodeById } from '../src/app/content/utils/archiveTreeUtils';
+import { ARCHIVE_URL, REACT_APP_ARCHIVE_URL, REACT_APP_OS_WEB_API_URL } from '../src/config';
+import books from '../src/config.books';
+import createArchiveLoader from '../src/gateways/createArchiveLoader';
+import createOSWebLoader from '../src/gateways/createOSWebLoader';
+import { Redirects } from '../src/redirects/types';
+
+const args = argv as any as {
+  bookId: string
+  newVersion: string | number;
+};
+
+const booksPath = path.resolve(__dirname, '../src/config.books.json');
+const redirectsPath = path.resolve(__dirname, '../src/redirects/');
```

Comment:
does it make any sense to use our `data` folder here so we don't overload the `src` directory?